src/main/kotlin/com/vk/modulite/highlighting/hints/BaseInlayHintsCollector.kt

```diff
@@ -71,7 +71,7 @@ abstract class BaseInlayHintsCollector(val project: Project, editor: Editor) : F
     }
 
     private fun singleModuliteNamePresentation(name: String, nameToShow: String): InlayPresentation {
-        val textPresentation = factory.text(nameToShow)
+        val textPresentation = factory.smallTextWithoutBackground(nameToShow)
```

Comment:
I disagree.
The [smallTextWithoutBackground](https://github.com/JetBrains/intellij-community/blob/7a1566b145b677302a256e9c4ed44e452e526443/platform/lang-impl/src/com/intellij/codeInsight/hints/presentation/PresentationFactory.kt#L54) and [text](https://github.com/JetBrains/intellij-community/blob/7a1566b145b677302a256e9c4ed44e452e526443/platform/lang-impl/src/com/intellij/codeInsight/hints/presentation/PresentationFactory.kt#L81) functions are organized differently.

It still seems to me that it is possible to use `container`. Or I'll make my own `textWithoutBackground`.

Reply:
Decided it was time-consuming